### PR TITLE
Design updates

### PR DESCRIPTION
**Pull request by skullface**
This PR continues the work from #73, including:

- homepage, Businesses, Maintainers, and user layouts;
- small layout and typography fixes for different screen sizes and new components. 

Ready for new creative assets!

cc @sophshep

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -64,15 +63,17 @@ img.img-responsive {
...
 
 .pquote-credit {
   margin-top: $spacer-3;
   font-size: 14px;
 }
 
-
+// User profile
+.bio-details span {
+  margin-right: $spacer-4;
```
Comment (sophshep) [inline app/assets/stylesheets/application.scss]: Instead of adding this as CSS, you can use a utility class: `mr-4` will give you the same thing.

**Comment by sophshep** [comment]
Some design feedback:

### Home

* The navigation could be given a wider container (`.container-lg`). This will give more breathing room to the links.
* On smaller screens, the "sign up with github" wraps:
image. This should be a block element so it jumps to the next line.
* The GitHub logo should be removed from each of the buttons (we usually don't put it in our sign up buttons) 
* Too much extra space here - we can remove the horizontal rule and let the bg colors separate the sections
* The form gets pretty squished on small screens:
image

### Business
* The Set A Reminder box here isn't prominent enough. What if it was white?
image

### User profile page
image

Don't forget to give this page some design love! You can get to it by hitting "sign in with github" and then accepting.

**Comment by skullface** [comment]
Thanks for the feedback, @sophshep! Updates made in the latest set of commits.

User profile layouts:

Logged-in user with some profile info filled out & some commit history:
open source friday - logged in user

"Empty" state of a user who isn't you, who hasn't yet joined, but has lots in their GitHub profile:
open source friday - empty user

**Comment by sophshep** [comment]
Welp, that didn't work! 

@MikeMcQuaid, can you give me (and @skullface!) full access to this repo so that we can make changes?

**Comment by MikeMcQuaid** [comment]
Sorry @sophshep. You have access now.

**Pull request merged**